server/channels/api4/system_test.go

```diff
@@ -83,12 +83,38 @@ func TestGetPing(t *testing.T) {
 		require.Contains(t, respString, "testvalue")
 	}, "ping feature flag test")
 
+	t.Run("ping root_status test", func(t *testing.T) {
+		if os.Geteuid() != 0 {
+			t.Skip("Skipping due to system user not having root privileges")
+		}
```

Comment:
1/5: I don't think that the test will ever be run as root. It's fine to just check if `root_status` is set for admins.

Reply:
We don't need to skip the test any longer, right?

Reply:
Yes, will remove that.

Reply:
Removed it via 81fa204cb49b18cf6947013bd60520e3670b2976 to move the PR forward